install.sh

```
@@ -33,6 +22,933 @@
 BIRDNET_GO_VERSION="nightly"
 BIRDNET_GO_IMAGE="ghcr.io/tphakala/birdnet-go:${BIRDNET_GO_VERSION}"
 
+# Logging configuration
+LOG_DIR="$HOME/birdnet-go-app/logs"
+# Generate timestamped log file name: install-YYYYMMDD-HHMMSS.log
+LOG_TIMESTAMP=$(date '+%Y%m%d-%H%M%S')
+LOG_FILE="$LOG_DIR/install-${LOG_TIMESTAMP}.log"
+
+# Initialize logging system early
+setup_logging
```

Shellcheck (0.10.0)

[error] This function is only defined later. Move the definition up.

(SC2218)